spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactory.java

```diff
@@ -46,4 +46,14 @@
 
 	void clearConnectionListeners();
 
+	/**
+	 * Return a separate connection for publishers (if implemented).
+	 * @return the connection.
+	 * @throws AmqpException an exception.
+	 * @since 2.0.2
+	 */
+	default Connection createPublisherConnection() throws AmqpException {
```

Comment:
I don't think we need this since we could just get direct access to the publisherCF delegate in the producers.

Reply:
See my comment above.